.gitignore

```diff
@@ -1 +1,2 @@
 /node_modules
+config.json
```

Comment:
i suggest you check in a file called "config.json.sample" and have the keys you are expecting set there, and a note to rename it to config.json, so users will know they have to do that

Reply:
Good idea, I will add that file.